__tests__/posts.ts

```diff
@@ -2016,4 +2014,20 @@ describe('mutation checkLinkPreview', () => {
     expect(res.data.checkLinkPreview.title).toEqual(sampleResponse.title);
     expect(res.data.checkLinkPreview.image).toEqual(defaultImage.placeholder);
   });
+
+  it('should return post by canonical', async () => {
```

Comment:
Perhaps also add a case for existing post to return id?

Reply:
Good catch! I think we should add it as another `expect` statement from every test case we found the link in the DB. Let me update these two.

Reply:
I've also added some `expect` statement on cases where we don't expect it 🚀